File: ComInLanWindows/ComInLan/Model/Packet/IServerPacket.cs
using ComInLan.Model.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ComInLan.Model.Packet
{
	public interface IServerPacket : IJson
	{
		string Id { get; }
		string DomainId { get; }
		string Name { get; }
		ServerPacketType Type { get; }
	}
}

File: ComInLanDroid/cominlan/src/main/java/com/onballgroup/cominlan/model/packet/IServerPacket.java
package com.onballgroup.cominlan.model.packet;

import com.onballgroup.cominlan.model.Base.IJson;

/**
 * Created by Phong Le on 4/17/2016.
 */
public interface IServerPacket extends IJson {
    String getId();
    String getDomainId();
    String getName();
    ServerPacketType getType();
}
